code/game/objects/items/weapons/soap.dm

```diff
@@ -12,11 +12,11 @@
 	var/key_data
 
 	var/list/valid_colors = list(COLOR_GREEN_GRAY, COLOR_RED_GRAY, COLOR_BLUE_GRAY, COLOR_BROWN, COLOR_PALE_PINK, COLOR_PALE_BTL_GREEN, COLOR_OFF_WHITE, COLOR_GRAY40, COLOR_GOLD)
-	var/list/valid_scents = list("fresh air", "cinnamon", "mint", "cocoa", "lavender", "an ocean breeze", "a summer garden", "vanilla", "cheap perfume")
-	var/list/scent_intensity = list("faintly", "strongly", "overbearingly")
+	var/list/valid_scents = list("fresh air", "cinnamon", "mint", "cocoa", "lavender", "an ocean breeze", "a summer garden", "vanilla", "cheap perfume","█████████","\[DATA EXPUNGED\]")
```

Comment:
The Linters hate the redacted white box... fix this

Reply:
is there a version of that unicode that gets through the linters?

Reply:
No

Reply:
alright i removed the linted text

Reply:
press "resolve conversation" when you resolve an issue

Reply:
kk